install/kubernetes/cilium/values.yaml

```
@@ -1005,6 +1005,11 @@ enableIPv6Masquerade: true
 egressGateway:
   enabled: false
 
+# -- Enables external VXLAN tunnel devices integration (beta) to allow
+# third party VXLAN devices talk to cilium pod over cilium tunnel directly.
+externalVxlan:
+  enabled: false
+
```

I would rework this section to include the related values so that they would be grouped logically:

```suggestion
  # -- A space separated list of external VXLAN tunnel device endpoint IPs, for example TODO
  endpoint: ~
  # -- A space separated list of external VXLAN tunnel device network CIDRs, for example TODO
  CIDR: ~
  # -- A space separated list of external VXLAN tunnel device MAC addresses (VTEP MAC), for example TODO
  MAC: ~
```

Then, adapt  `install/kubernetes/cilium/templates/cilium-configmap.yaml` accordingly (the comments can be removed from there since they would be now documented in `values.yaml` and `README.md`).

---

@kaworu thank you for your careful review, again,  I am completely new to helm, so I may not completely understand your suggestion, I noticed github has the  `Commit suggestion`  with your new changes, I wonder what it does and if I can click `Commit suggestion`, or do you prefer I modify the changes locally as you suggested and push the new changes, I am just worried I may not understand your suggestions :). from learning purpose, I can change locally and push the changes.

---

@kaworu I made local changes and pushed the changes, Could you review if I did it right

---

@vincentmli updating your commit is best, thanks for doing it that way! Helm patch LGTM now.